arviz/stats/diagnostics.py

```diff
@@ -826,13 +826,15 @@ def _mcse_sd(ary):
     ary = np.asarray(ary)
     if _not_valid(ary, shape_kwargs=dict(min_draws=4, min_chains=1)):
         return np.nan
-    ess = _ess_sd(ary)
+    ary = ary - ary.mean()
+    ess = _ess_mean(abs(ary))
+    evar = (ary**2).mean()
+    varvar = ((ary**4).mean() - evar**2) / ess
+    varsd = varvar / evar / 4
     if _numba_flag:
-        sd = np.float(_sqrt(svar(np.ravel(ary), ddof=1), np.zeros(1)))
+        mcse_sd_value = np.float(_sqrt(np.ravel(varsd), ddof=1), np.zeros(1))
```

Comment:
```suggestion
        mcse_sd_value = np.float(_sqrt(np.ravel(varsd), np.zeros(1)))
```

Reply:
Or this should be the same as before

`np.float(_sqrt(svar(np.ravel(varsd), ddof=1), np.zeros(1)))`